tests/backup/backup_multiple_provisioner_test.go

```diff
@@ -0,0 +1,195 @@
+package tests
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	"github.com/pborman/uuid"
+	api "github.com/portworx/px-backup-api/pkg/apis/v1"
+	"github.com/portworx/torpedo/drivers/backup"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/pkg/log"
+	. "github.com/portworx/torpedo/tests"
+	"math/rand"
+)
+
+// This MultipleProvisionerCsiSnapshotDeleteBackupAndRestore testcase to test restore of namespaces with multiple provisioners using backup when snapshot is deleted
+var _ = Describe("{MultipleProvisionerCsiSnapshotDeleteBackupAndRestore}", func() {
+
+	var (
+		restoreNames                              []string
+		scheduledAppContexts                      []*scheduler.Context
+		cloudCredName                             string
+		cloudCredUID                              string
+		backupLocationUID                         string
+		backupLocationName                        string
+		backupLocationMap                         map[string]string
+		providers                                 []string
+		schedulePolicyName                        string
+		schedulePolicyUID                         string
+		scheduleUid                               string
+		srcClusterUid                             string
+		schedulePolicyInterval                    = int64(15)
+		allAppContext                             []*scheduler.Context
+		scheduleList                              []string
+		randomStringLength                        = 10
+		appSpecList                               []string
+		scheduledAppContextsForMultipleAppSinleNs []*scheduler.Context
+		multipleProvisionerSameNsScheduleName     string
+		multipleNsSchBackupName                   string
+		clusterProviderName                       = GetClusterProvider()
+		provisionerDefaultSnapshotClassMap        = GetProvisionerDefaultSnapshotMap(clusterProviderName)
+	)
+
+	JustBeforeEach(func() {
+		StartPxBackupTorpedoTest("MultipleProvisionerCsiSnapshotDeleteBackupAndRestore", "Delete Csi snapshot and restore namespaces from backup", nil, 296725, Sn, Q4FY24)
+
+		backupLocationMap = make(map[string]string)
+		providers = GetBackupProviders()
+
+		// Deploy application for Default backup
+		appSpecList = []string{"postgres"}
```

Comment:
Why is this hard-coded here?

Reply:
Reading from the current applist may not work , will need to impl getting only the prefix will discuss with you